### PR TITLE
implement the special handling for avogadro constant

### PR DESCRIPTION
**Pull request by exaexa**
@paulflang can you check if this would do the trick as discussed on slack?

### CODE REVIEW

**Comment by exaexa** [comment]
PS can you recommend a SBML test case that would contain this thing?

**Comment by paulflang** [comment]
Thanks, looks good to me. I cannot recommend a particular test case. I guess any of those you grepped in you Slack message will be fine.

File: src/structs.jl [hunk 1]
```diff
@@ -139,6 +139,19 @@ end
 """
 $(TYPEDEF)
 
+A special value representing the avogadro constant (which is a special named
```
Comment (giordano) [inline src/structs.jl:142]:
```suggestion
A special value representing the Avogadro constant (which is a special named
```
:wink:

**Review by giordano — APPROVED** [review]
Lovely

File: src/structs.jl (hunk 1 above)
Comment (exaexa) [reply src/structs.jl:142]: image
tho.

**Pull request merged**